README.md

```diff
@@ -173,19 +178,264 @@ The explanation functions accept both models and pipelines as input. If a model
 * Permutation Feature Importance Explainer: Permutation Feature Importance is a technique used to explain classification and regression models that is inspired by [Breiman's Random Forests paper](https://www.stat.berkeley.edu/%7Ebreiman/randomforest2001.pdf) (section 10). At a high level, it works by randomly shuffling data one feature at a time for the entire dataset and calculating how much the performance metric of interest decreases. The larger the change, the more important that feature is.
 * LIME Explainer: LIME Explainer uses the state-of-the-art Local interpretable model-agnostic explanations [(LIME)](https://github.com/marcotcr/lime) algorithm to create local surrogate models. Unlike the global surrogate models, LIME focuses on training local surrogate models to explain individual predictions.
 
-Tabular Explainer: Used with tabular datasets, it currently employs the following logic to invoke the direct SHAP explainers:
+* Tabular Explainer: Used with tabular datasets, it currently employs the following logic to invoke the direct SHAP explainers:
 
 
-| Original Model   | Invoked Explainer  |
-|-----|-----|
-| Tree-based models | SHAP TreeExplainer|
-| Deep Neural Network models| SHAP DeepExplainer|
-| Linear models | SHAP LinearExplainer |
-| None of the above  | SHAP KernelExplainer |
+  | Original Model   | Invoked Explainer  |
+  |-----|-----|
+  | Tree-based models | SHAP TreeExplainer|
+  | Deep Neural Network models| SHAP DeepExplainer|
+  | Linear models | SHAP LinearExplainer |
+  | None of the above  | SHAP KernelExplainer |
 
 <a name=Example></a>
 
-# Example
+# Use Interpret-Community
+
+
+## Interpretability in training
+
+
+1. Train your model in a local Jupyter notebook.
```

Comment:
Great job!

We've overloaded the word local and need to clarify so users don't get confused. In this content it's local compute, ie laptop.
Further down, we have a distinction for global vs local feature importance. We could say that local measures focus on the contribution of features for a specific prediction, whereas global measures take all predictions into account.
  
Perhaps change this instance to something like:
Train your model in a Jupyter notebook running on your local laptop.
And then note an alternative may be on a cloud.  
Then further down, we can highlight the difference between global and local feature importance.

Reply:
Very great points. Applied.